src/spikeinterface/postprocessing/principal_component.py

```diff
@@ -120,25 +156,26 @@ def get_pca_model(self):
             pca_models = self.data[f"pca_model_{mode}"]
         return pca_models
 
-    def get_projections_one_unit(self, unit_id, sparse=False):
+    def get_projections_one_unit(self, unit_id, sparse=False, kept_projections=None, kept_spikes=None):
```

Comment:
same comment that waveforms.
This new signature is a bit allien and use for the merge which should be internal

Reply:
Ok, but let's discuss what do do, because as said, we would nee to provide sliced waveforms/spikes to be efficient